stable/fluent-bit/templates/config.yaml

```diff
@@ -47,7 +52,7 @@ data:
         Max_Entries     {{ .Values.input.systemd.maxEntries }}
         Read_From_Tail  {{ .Values.input.systemd.readFromTail }}
 {{- end }}
-{{ .Values.extraEntries.input | indent 8 }}
+{{ .Values.extraEntries.input | indent 4 }}
```

Comment:
This is broken. Please revert.

Reply:
Actually, I think it was also slightly broken when we let in the systemd.enabled patch... it really should be right after the end of '{{- if .Values.trackOffsets }}'

Reply:
Ok.  Cool.  I just made that same comment before I saw yours.  I can correct these in the PR.